tests/Agent/IntegrationTests/UnboundedIntegrationTests/Redis/StackExchangeRedisTests.cs

```diff
@@ -42,9 +42,6 @@ public StackExchangeRedisTestsBase(TFixture fixture, ITestOutputHelper output) :
                 exerciseApplication: () => _fixture.AgentLog.WaitForLogLines(AgentLogBase.TransactionTransformCompletedLogLineRegex, TimeSpan.FromMinutes(2), 2)
             );
 
-            // Confirm both transaction transforms have completed before moving on to host application shutdown, and final sendDataOnExit harvest
```

Comment:
Did you mean to remove this?

Reply:
Yeah, it was a merge error from my last PR. It's duplicated code.

Reply:
Got it, thanks!